code/modules/shuttle/on_move.dm

```diff
@@ -47,33 +280,65 @@
 		else
 			shake_camera(src, 7, 1)
 
-/mob/living/carbon/onShuttleMove(turf/T1, rotation, knockdown = TRUE)
+/mob/living/carbon/afterShuttleMove(list/movement_force)
 	. = ..()
-	if(!.)
```

Comment:
the parent is different now. This used to be checking to make sure the mob had actualy been moved but that's no longer necesary as afterShuttleMove only gets called on things that have been moved anyway.